Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -297,11 +297,15 @@ CGameListCtrl::~CGameListCtrl()
 
 template <typename T>
 static void InitBitmap(wxImageList* img_list, std::vector<int>* vector, wxWindow* context,
-                       const wxSize& usable_size, T index, const std::string& name)
+                       const wxSize& usable_size, T index, const std::string& name,
+                       bool themed = false)
 {
   wxSize size = img_list->GetSize();
-  (*vector)[static_cast<size_t>(index)] = img_list->Add(WxUtils::LoadScaledResourceBitmap(
-      name, context, size, usable_size, WxUtils::LSI_SCALE | WxUtils::LSI_ALIGN_VCENTER));
+  (*vector)[static_cast<size_t>(index)] = img_list->Add(
+      themed ? WxUtils::LoadScaledThemeBitmap(name, context, size, usable_size,
+                                              WxUtils::LSI_SCALE | WxUtils::LSI_ALIGN_VCENTER) :
+               WxUtils::LoadScaledResourceBitmap(name, context, size, usable_size,
+                                                 WxUtils::LSI_SCALE | WxUtils::LSI_ALIGN_VCENTER));
```

Comment:
I think it would be nice to move the whole ternary expression to a separate line (and assign it to a variable) and then just have `(*vector)[static_cast<size_t>(index)] = img_list->Add(bitmap);` at the end.